app/Models/Accessory.php

```diff
@@ -329,11 +329,24 @@ public function getEula()
     }
 
 
+    /**
+     * Check how many items within an accessory are checked out
+     *
+     * @author [A. Gianotto] [<snipe@snipe.net>]
+     * @since [v5.0]
+     * @return int
+     */
+    public function numCheckedOut()
+    {
+        return $this->users_count ?? $this->users->count();
```

Comment:
I think for this you want `->users()->` otherwise you're loading the actual users up instead of running the `SELECT COUNT(*) FROM ...` query, which we would prefer.